expo/src/Table.js

```diff
@@ -10,9 +10,11 @@ import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
 import './App.css';
 import './Table.css';
 
+import './customize/customize';
```

Comment:
Is this an extra import, or is it supposed to be the css?

Reply:
So in table.js there was an array of colors so I took that out and put that in customize.js and that's what the import is for